youtube_dl/extractor/americastestkitchen.py

```diff
@@ -90,3 +92,74 @@ def _real_extract(self, url):
             'series': try_get(episode, lambda x: x['show']['title']),
             'episode': episode.get('title'),
         }
+
+
+class AmericasTestKitchenSeasonIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?(?P<show>americastestkitchen|cookscountry)\.com/episodes/browse/season\_(?P<id>\d+)'
+    _TESTS = [{
+        # ATK Season
+        'url': 'https://www.americastestkitchen.com/episodes/browse/season_1',
+        'info_dict': {
+            'id': 'season-1',
+            'title': 'Season 1',
+        },
+        'playlist_count': 13,
+    }, {
+        # Cooks Country Season
+        'url': 'https://www.cookscountry.com/episodes/browse/season_12',
+        'info_dict': {
+            'id': 'season-12',
+            'title': 'Season 12',
+        },
+        'playlist_count': 13,
+    }, {
+        # Multi-digit season
+        'url': 'https://www.americastestkitchen.com/episodes/browse/season_20',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        show_name, season = re.match(self._VALID_URL, url).groups()
+
+        slug = 'atk' if show_name == 'americastestkitchen' else 'cco'
+
+        filters = [
+            'search_season_list:Season %s' % season,
+            'search_document_klass:episode',
+            'search_show_slug:%s' % slug,
+        ]
+
+        params = {
+            'facetFilters': json.dumps(filters),
+            'attributesToRetrieve': 'search_url',
+            'attributesToHighlight': '',
```

Comment:
not needed.

Reply:
This is actually required to remove `_highlightResult` from the response, which is fairly large